React/Views/UIView+React.m

```diff
@@ -287,19 +286,23 @@ - (void)reactFocus
 #else
 	if (![self becomeFirstResponder]) {
 #endif // TODO(macOS GH#774)]
-		self.reactIsFocusNeeded = YES;
-	}
+// [TODO(macOS)
+    _pendingFocusView = self;
+  } else {
+    _pendingFocusView = nil;
+  }
+// TODO(macOS)]
```

Comment:
I should have clarified. The macOS tags are of the following forms. The Github Issue in the tag is either the generic #774 or a more specific one if we have it / want to actually upstream the fix.

```
# One line change
// TODO(macOS GH#774)

#Block change
// [TODO(macOS GH#774)
...
// ]TODO(macOS GH#774)
```

Reply:
made new issue. Added a test page.